action.yml

```diff
@@ -80,10 +80,102 @@ inputs:
     description: 'Postremove script location.'
     default: ''
     required: false
+  nfpm_destination:
+    description: "Where to install the nFPM binary (default: $HOME/bin/nfpm)"
+    type: string
+    default: "$HOME/bin/nfpm"
+  nfpm_version:
+    description: "The version of nFPM to install (default: latest)"
+    type: string
+    default: Latest
 
-# action runner (golang:latest image)
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
+  using: composite
   env:
-    GO111MODULE: 'on'
+    INPUT_NAME: ${{ inputs.name }}
+    INPUT_ARCH: ${{ inputs.arch }}
+    INPUT_VERSION: ${{ inputs.version }}
+    INPUT_MAINTAINER: ${{ inputs.maintainer }}
+    INPUT_VENDOR: ${{ inputs.vendor }}
+    INPUT_DESCRIPTION: ${{ inputs.description }}
+    INPUT_HOMEPAGE: ${{ inputs.homepage
+    INPUT_LICENSE: ${{ inputs.license }}
+    INPUT_DEPENDS: ${{ inputs.depends }}
+    INPUT_BINARY: ${{ inputs.binary }}
+    INPUT_BIN_PATH: ${{ inputs.bin_path }}
```

Comment:
What is the `INPUT_BIN_PATH`? I'm comparing it to the list of inputs from the README and this looks like it's new.

Reply:
I copied it directly from the previous docker vars and missed that it was no longer an input.

Reply:
I was actually wrong, it's used by the template generator to set the package destination. I fixed it in my latest commit.